frontend/src/lang/es.ts

```diff
@@ -281,7 +281,9 @@ export const es: LanguageConfig = {
             li0:
               "Quédese en casa cuando esté enfermo, excepto para recibir atención médica.",
             li1:
-              "Autoaíslese durante 10 días completos después de que aparezcan los síntomas (o a partir del día siguiente a la realización de la prueba, si no tiene síntomas).",
+              "Quédate en casa durante 5 días." +
```

Comment:
missing space after periods, maybe, idk if thats the case for spanish

![image](https://user-images.githubusercontent.com/4952042/149361894-8856052d-446d-4cd5-8b15-fa3969571888.png)